tests/common/graph/test_graph_pattern.py

```diff
@@ -213,3 +213,7 @@ def test_join_pattern_with_special_input_node():
             ref_pattern.add_edge(node, added_node)
 
     assert pattern == ref_pattern
+
+
+def test_dump(tmp_path):
+    TestPattern.first_pattern.dump_graph(tmp_path / "pattern.dot")
```

Comment:
What does this test check?

Reply:
That dump_graph function is not failed with AttributeError.
Added check that `pattern.dot` exist.